backend.py

```diff
@@ -295,6 +298,20 @@ def main():
     except Exception as e:
         logger.error(f"Failed to set sentry context: {e}")
 
+    if not manufacturing_config.get("enable", True) and manufacturing_config.get(
```

Comment:
Please dont put complex logic here. Move it to a managers init() function please. E.g. create an SSHManager.init()